style.css

```diff
@@ -377,3 +374,237 @@ fieldset {
   right: 7rem;
   margin-top: -50px;
 }
+
+.nav-link {
+  display: none;
+}
+
+.ruby {
+  display: none;
+}
+
+.el {
+  display: initial;
+}
+
+.about-1 {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+@media screen and (min-width: 768px) {
+  * {
+    overflow-x: hidden;
+  }
+
+  header {
+    padding: 1rem 5rem;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    background-color: white;
+  }
+
+  header .menu {
+    display: none;
+  }
+
+  .nav-link {
+    display: initial;
+  }
+
+  .nav-link ul {
+    display: flex;
+    align-items: center;
+  }
+
+  .nav-link li a {
+    font-size: 1.3rem;
+    padding-left: 5rem;
+    color: #344563;
+  }
+
+  .section-1 {
+    padding: 0 10rem 0 5rem;
+    width: 80vw;
+    margin: 0 auto;
+    background: url(image/Header\ bg.svg);
+    border-bottom-left-radius: 100px;
+    background-color: white;
+    margin-bottom: 5rem;
+    margin-top: 4rem;
+  }
+
+  .section-1 h1 {
+    font-size: 3rem;
+  }
+
+  .section-1 p {
+    font-size: 1.5rem;
+  }
+
+  .Works {
+    width: 100%;
+    margin: auto;
+  }
+
+  .card {
+    display: flex;
+    padding: 1.5rem;
+    width: 80vw;
+    margin: 2rem auto;
+  }
+
+  .card:hover {
+    border: 1px solid #396df2;
+    box-shadow: 0 0 1px;
+    cursor: pointer;
+  }
+
+  .profile {
+    overflow: hidden;
+    width: 50%;
+    padding: 0 1.5rem 0 0;
+  }
+
+  .profile img {
+    object-fit: contain;
+  }
+
+  .card-description {
+    max-width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 1.5rem 0 0 1.5rem;
+    gap: 0.5rem;
+  }
+
+  .card-description h3 {
+    font-size: 2rem;
+    color: #091e42;
+  }
+
+  .profile-description h5 {
+    font-size: 1.3rem;
+    font-weight: 500;
+  }
+
+  .paragraph-description p {
+    font-size: 2vmin;
+    line-height: 1.5;
+  }
+
+  .ruby {
+    display: initial;
+    white-space: nowrap;
+  }
+
+  .card-1 {
+    order: 3;
+  }
+
+  .card-2 {
+    order: 4;
+  }
+
+  .card-3 {
+    order: 1;
+  }
+
+  .card-4 {
+    order: 2;
+  }
+
+  .order-1 {
+    order: 2;
+    padding-left: 1rem;
+  }
+
+  .order-2 {
+    order: 1;
+  }
+
+  #About {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 5rem 9rem;
+    margin: auto;
+    align-items: center;
+  }
+
+  .about-1 {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    gap: 1rem;
+  }
+
+  .about-2 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+  }
```

Comment:
As per the Figma template of your choosing, these two sections are not of equal width. The first `about section` is about 40% and the other about section is about 60%. I have recorded [this video](https://www.loom.com/share/f5e07483e641494b84a8d37b17b2fa51) where I compared your UI with that of the Figma template and explained the fix for it. Kindly fix this issue and then submit it for a re-review.